### ISSUE TITLE
[Suggestion] Add the Innie guns to the smugglers.

### ISSUE THREAD

**Issue by DronzTheWolf** [HaloSpaceStation/HaloSpaceStation13#2384]
**Suggestion Details**

The innie guns, out-of-date surplus MA3s an MA37s, are unusually absent from the smuggler lists. Unlike the brand-spanking-new UNSC guns like the DMR, MA5B, and M90 shotguns. I say we add the applicable innie guns to the smugglers. The Anti-Materiel Rifle is already in, and I think we all agree the innie DMR shouldn't be in since it's an innie modification, but the MA3 and MA37 are both old surplus guns that could be grabbed by smugglers and innies alike. The innie LMG can probably also be added, since the UNSC and base SS13 ones are available.

**Comment by caelaislinn** [comment]
Great idea, the purchasable guns are balanced around what you can unlock via missions but they should be obtainable somewhere if they aren't already.

**Comment by DronzTheWolf** [comment]
Yeah, the Geminus gun smugglers being able to get their hands on brand-spanking-new UNSC guns and not these old ones is kinda dumb to me. If anything, the newer/better UNSC guns should be more rare. But we're still able to get BRs, LMGs, MA5Bs, and pretty much everything but the big power weapons like the SRS and RPG.

**Comment by BDpuffy420** [comment]
was done
